index.html

```diff
@@ -177,14 +177,35 @@ <h3 class="Project-title">
                         <span class="last-child">Ruby</span>
                     </div>
                     <div class="See-Project">
-                        <button>
+                        <button class="btn-primary">
                             See Project
                         </button>
                     </div>
                 </div>
             </section>
-
         </section>
+
+        <footer class="footer">
+            <section class="Contact-us">
+                    <h1 class="Section-title">
+                        CONTACT US
+                    </h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                    </p>
+                    <form class="Contact-us__Form" id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="#" method="post">
```

Comment:
-After submit the form i have this error message:
![Screen Shot 2021-09-21 at 16 18 18](https://user-images.githubusercontent.com/22258274/134249096-10dd6405-a3c1-475a-a1fe-16c21cba151c.png)

- Remember that your `` action `` value should be similar to `` https://formspree.io/f/{form_id} `` .

Reply:
I removed it because it does a linter error and added so, you can see it  says "{}" is not supported in HTML checker